eglib/configure.ac

```diff
@@ -140,11 +140,11 @@ AC_CHECK_FUNCS(fork execv execve)
 
 AC_ARG_WITH([overridable-allocators], [  --with-overridable-allocators	allow g_*alloc/g_free to call custom allocators set via g_mem_set_vtable])
 
-if test x$with_overridable_allocators == xyes; then
-	AC_DEFINE(ENABLE_OVERRIDABLE_ALLOCATORS,1,[Overridable allocator support enabled])
-  AC_MSG_NOTICE([Overridable allocator support enabled])
-else
+if test x$with_overridable_allocators == xno; then
   AC_MSG_NOTICE([Overridable allocator support disabled])
+else
+  AC_DEFINE(ENABLE_OVERRIDABLE_ALLOCATORS,1,[Overridable allocator support enabled])
+  AC_MSG_NOTICE([Overridable allocator support enabled])
 fi
```

Comment:
Once I port it to linux and address the issues raised here we can benchmark this plus the tags to see how worse it gets.